opre_ops/opre_ops/settings/cloudgov.py

```diff
@@ -45,7 +43,7 @@ def get_json_env_var(variable_name):
 }
 
 # SECURITY: Keep the secret keys used in production secret!
-SECRET_KEY = user_provided_env['DJANGO_SECRET_KEY']
+SECRET_KEY = env.get_credential('APP_SECRET_KEY', generate_random_string(50))
```

Comment:
[question] Are we setting `'APP_SECRET_KEY'` as user-provided environment variable on cloud.gov?  If not, it may not be kept between deployment, and is that okay if we always use a random string?

Reply:
yeah this is something that will need to be addressed in the future ticket (I mentioned on Slack I ripped out what Alex had previously implemented here but it was incomplete so it was complicating this work).